src/test/java/org/opensearch/neuralsearch/query/HybridQueryIT.java

```diff
@@ -692,6 +698,101 @@ public void testWrappedQueryWithFilter_whenIndexAliasHasFilters_thenSuccess() {
         }
     }
 
+    @SneakyThrows
+    public void testConcurrentSearchWithMultipleSlices_whenSingleShardIndex_thenSuccessful() {
```

Comment:
Do not review this class as it already merged in main. The change is showing here because we have cherry picked the commit.